lib/Runtime/Base/FunctionBody.h

```diff
@@ -2690,6 +2693,23 @@ namespace Js
         bool GetHasRestParameter() const { return (flags & Flags_HasRestParameter) != 0; }
         void SetHasRestParameter() { SetFlags(true, Flags_HasRestParameter); }
 
+        bool NeedScopeObjectForArguments(bool hasNonSimpleParams)
+        {
+            Assert(HasReferenceableBuiltInArguments());
+            
```

Comment:
Done.

---
In reply to: [67793826](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r67793826) [](ancestors = 67793826)